src/utilities/device_utils.c

```diff
@@ -3078,3 +3122,91 @@ hypre_bind_device( HYPRE_Int myid,
 {
    return hypre_bind_device_id(-1, myid, nproc, comm);
 }
+
+/*--------------------------------------------------------------------------
+ * Some MPI_Comm calls specifically for device
+ * 
+ *--------------------------------------------------------------------------*/
+#if defined(HYPRE_USING_GPU) || defined(HYPRE_USING_DEVICE_OPENMP)
+#if defined(HYPRE_SEQUENTIAL)
+HYPRE_Int hypre_MPI_Comm_split_type( hypre_MPI_Comm comm, HYPRE_Int split_type, HYPRE_Int key,
+                                     hypre_MPI_Info info, hypre_MPI_Comm *newcomm )
+{
+   HYPRE_UNUSED_VAR(comm);
+   HYPRE_UNUSED_VAR(split_type);
+   HYPRE_UNUSED_VAR(key);
+   HYPRE_UNUSED_VAR(info);
+   HYPRE_UNUSED_VAR(newcomm);
+   return (0);
+}
+
+HYPRE_Int hypre_MPI_Info_create( hypre_MPI_Info *info )
+{
+   HYPRE_UNUSED_VAR(info);
+   return (0);
+}
+
+HYPRE_Int hypre_MPI_Info_free( hypre_MPI_Info *info )
+{
+   HYPRE_UNUSED_VAR(info);
+   return (0);
+}
+#else
+HYPRE_Int
+hypre_MPI_Comm_split_type( hypre_MPI_Comm comm, HYPRE_Int split_type, HYPRE_Int key,
+                           hypre_MPI_Info info, hypre_MPI_Comm *newcomm )
+{
+   return (HYPRE_Int) MPI_Comm_split_type(comm, split_type, key, info, newcomm );
+}
+
+HYPRE_Int
+hypre_MPI_Info_create( hypre_MPI_Info *info )
+{
+   return (HYPRE_Int) MPI_Info_create(info);
+}
+
+HYPRE_Int
+hypre_MPI_Info_free( hypre_MPI_Info *info )
+{
+   return (HYPRE_Int) MPI_Info_free(info);
+}
+#endif
+#endif
```

Comment:
Just chiming in... I think these should probably go back to `mpistubs.c`. I see they were just being compiled for GPUs before, but I think it's fine to compile them always and just place together with all the other MPI wrapper calls

Reply:
Thanks @victorapm , I made the changes in commit [7163b69](https://github.com/hypre-space/hypre/pull/1423/commits/7163b696df273d3365c0a3b1980efaad2633fba4)